extensions/HyperVExtension/src/HyperVExtension/Strings/en-US/Resources.resw

```diff
@@ -158,19 +158,19 @@
     <comment>Label text to display in front of the users current checkpoint name. </comment>
   </data>
   <data name="DownloadFileFound" xml:space="preserve">
-    <value>File {0} already exists. We'll use this file to create the virtual machine</value>
+    <value>File {0} already exists. We'll use this file to create the virtual machine.</value>

```

Comment:
NIT `File {0} exists`

Reply:
I think your suggestion would make sense if we had just created the file, but I'm reading this as it "already" existed before we did whatever we just did, so I think the "already" is more clear. I could be misunderstanding though.